go.mod

```diff
@@ -260,11 +265,13 @@ replace (
 	github.com/flynn/go-shlex => github.com/flynn/go-shlex v0.0.0-20150515145356-3f9db97f8568
 	github.com/fogleman/gg => github.com/fogleman/gg v1.2.1-0.20190220221249-0403632d5b90
 	github.com/fsnotify/fsnotify => github.com/fsnotify/fsnotify v1.4.7
+	github.com/garyburd/redigo => github.com/garyburd/redigo v0.0.0-20150301180006-535138d7bcd7
```

Comment:
this is a sunset repository used only by cadvisor redis storage. we should find a way to isolate storage and container integrations in cadvisor so we don't pull this into our dependency tree if we don't need to